src/components/Tenants/AppSettings/PermissionsSettings/EnforceModeSwitch/styles.module.scss

```
@@ -0,0 +1,15 @@
+
+.switch {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    gap: mui.$mui-unit;
+    width: 100%;
+}
+
+.active {
+    color: var(--m3-primary, #a5000a);
+    // Selected mode must not rely on colour alone (a11y / admin UI rules).
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)